src/voku/helper/HtmlDomParser.php

```diff
@@ -55,6 +55,26 @@ class HtmlDomParser extends AbstractDomParser
         '{{',
     ];
 
+    /**
+     * The properties specified for each special script tag is an array of the following format:
+     * string script tag => regex script tag
+     *
+     * ```php
+     * protected $specialScriptTags = [
+     *     'text/html' => 'text\/html',
+     *     'text/x-custom-template' => 'text\/x-custom-template',
+     *     'text/x-handlebars-template' => 'text\/x-handlebars-template'
+     * ]
+     * ```
+     *
+     * @var string[]
+     */
+    protected $specialScriptTags = [
+        'text/html' => 'text\/html',
```

Comment:
Maybe we can use "preg_quote()" + instead of this duplicate key⇒value pair? What do you think?

Reply:
Yes, I think so its better to use preg_quote. Thanks.

Reply:
Thanks for your contribution. :+1: